src/main/java/moze_intel/projecte/network/packets/KeyPressPKT.java

```diff
@@ -50,34 +55,43 @@ public IMessage onMessage(KeyPressPKT message, MessageContext ctx)
 			
 				if (boots != null && boots.getItem() == ObjHandler.gemFeet)
 				{
-					GemArmor.toggleStepAssist(boots, player);
+					GemFeet.toggleStepAssist(boots, player);
 				}
 			}
 		}
 		
-		if (stack == null)
+		if (stack == null || !(stack.getItem() instanceof ItemPE))
```

Comment:
Should i leave it like this or change it back. People might derp and press C while on another item and blow up their base or something.